policies/templates/gcp_vpc_sc_ensure_access_levels_v1.yaml

```diff
@@ -64,16 +74,38 @@ spec:
              	lib.has_field(asset, "service_perimeter")
              
              	lib.get_constraint_params(constraint, params)
-             	required_access_levels_array := lib.get_default(params, "required_access_levels", [])
-             	required_access_levels := {p | p = required_access_levels_array[_]}
+             	mode := lib.get_default(params, "mode", "require")
              
              	perimeter_access_levels_raw := {split(r, "/") | r = asset.service_perimeter.status.access_levels[_]}
              	perimeter_access_levels := {r[3] | r = perimeter_access_levels_raw[_]}
              
-             	count(perimeter_access_levels - required_access_levels) != count(perimeter_access_levels) - count(required_access_levels)
+             	# For compatibility reasons, we support the old key name required_access_levels
+             	configured_access_levels := cast_set(lib.get_default(params, "access_levels", lib.get_default(params, "required_access_levels", [])))
```

Comment:
required_access_levels param is not defined

Reply:
@joecheuk That's intentional. No new user of this constraint should be using that field. It was defined in the previous version of this constraint. This is an old backwards compatibility check in case someone updated this template without updating their parameters.